showcase/app/templates/components/advanced-table.hbs

```diff
@@ -0,0 +1,1712 @@
+              </div>
+            </Hds::AdvancedTable::Td>
+            <Hds::AdvancedTable::Td @align="center">
+              <div {{style display="inline-block"}}>
+                <Hds::Button @size="small" @icon="plus" @text="Lorem ipsum" @color="secondary" />
+              </div>
+            </Hds::AdvancedTable::Td>
+            <Hds::AdvancedTable::Td @align="right">
+              <div {{style display="inline-block"}}>
+                <Hds::Button @size="small" @icon="plus" @text="Lorem ipsum" @color="secondary" />
+              </div>
+            </Hds::AdvancedTable::Td>
+          </div>
+          <div class="hds-advanced-table__tr">
+            <Hds::AdvancedTable::Th>Dropdown (with <code>@isInline</code>)</Hds::AdvancedTable::Th>
+            <Hds::AdvancedTable::Td @align="left">
+              <Hds::Dropdown @isInline={{true}} @listPosition="bottom-right" as |dd|>
+                <dd.ToggleIcon @icon="more-horizontal" @text="Overflow Options" @hasChevron={{false}} @size="small" />
+                <dd.Interactive @route="components.table">Dropdown</dd.Interactive>
+              </Hds::Dropdown>
+            </Hds::AdvancedTable::Td>
+            <Hds::AdvancedTable::Td @align="center">
+              <Hds::Dropdown @isInline={{true}} @listPosition="bottom-right" as |dd|>
+                <dd.ToggleIcon @icon="more-horizontal" @text="Overflow Options" @hasChevron={{false}} @size="small" />
+                <dd.Interactive @route="components.table">Dropdown</dd.Interactive>
+              </Hds::Dropdown>
+            </Hds::AdvancedTable::Td>
+            <Hds::AdvancedTable::Td @align="right">
+              <Hds::Dropdown @isInline={{true}} @listPosition="bottom-right" as |dd|>
+                <dd.ToggleIcon @icon="more-horizontal" @text="Overflow Options" @hasChevron={{false}} @size="small" />
+                <dd.Interactive @route="components.table">Dropdown</dd.Interactive>
+              </Hds::Dropdown>
+            </Hds::AdvancedTable::Td>
+          </div>
+          <div class="hds-advanced-table__tr">
+            <Hds::AdvancedTable::Th>Tooltip(Button)</Hds::AdvancedTable::Th>
+            <Hds::AdvancedTable::Td @align="left">
+              <Hds::TooltipButton @text="Hello!">
+                December 15, 2022,<br />16:40:09 PM
+              </Hds::TooltipButton>
+            </Hds::AdvancedTable::Td>
+            <Hds::AdvancedTable::Td @align="center">
+              <Hds::TooltipButton @text="Hello!">
+                December 15, 2022,<br />16:40:09 PM
+              </Hds::TooltipButton>
+            </Hds::AdvancedTable::Td>
+            <Hds::AdvancedTable::Td @align="right">
+              <Hds::TooltipButton @text="Hello!">
+                December 15, 2022,<br />16:40:09 PM
+              </Hds::TooltipButton>
+            </Hds::AdvancedTable::Td>
+          </div>
+        </div>
+      </div>
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Flex @label="Vertical alignment" as |SF|>
```

Comment:
Question: I think the vertical alignment got a little wonky along the way.... maybe the grid styles affected it? Looking at the showcase, the CSS property is being added but the top aligned example doesn't look right. What is expected with this argument?
<img width="1122" alt="Screenshot 2024-12-19 at 2 23 37 PM" src="https://github.com/user-attachments/assets/97430f38-ec85-4ed9-b879-c9ea29697163" />

Reply:
Looking into vertical align more, it doesnt work because advanced table doesn't use the table elements. Should we just remove this argument for now?